src/index.html

```diff
@@ -13,7 +13,16 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
+  <div class="container">
+    <form action="#" class="search" data-qa="big">
+      <img src="/images/Search.svg" alt="" class="search__img search__img--big">
+      <input type="text" class="search-bar search-bar--big" placeholder="Try “Los Angeles”" data-qa="keypress">
```

Comment:
It rather `search__input` then `search__bar`. Also, this is an element of the block then it must be 
```suggestion
      <input type="text" class="search__input search__input--big" placeholder="Try “Los Angeles”" data-qa="keypress">
```

Reply:
better to write `_` for modification instead of `--`